SYCL/Sampler/unnormalized-clamp-linear.cpp

```diff
@@ -0,0 +1,166 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
+
+// GPU does not correctly interpolate when using clamp.  Waiting on fix.
```

Comment:
Is it possible to run the test but XFAIL it?

Reply:
Done.